index.d.ts

```diff
@@ -0,0 +1,79 @@
+declare module "discord-webook-node" {
+  export type WebhookOptions = {
+    url: string;
+    throwErrors?: boolean;
+    retryOnLimit?: boolean;
+  };
+  export class Webhook {
+    constructor(options: string | WebhookOptions);
+    setUsername(username: string): void;
+    setAvatar(avatar: string): void;
+    sendFile(filePath: string): Promise<void>;
+    send(message: MessageBuilder): Promise<void>;
+    info(
+      title: string,
+      fieldName: string,
+      fieldValue: string,
+      inline: boolean
+    ): void;
+    warning(
+      title: string,
+      fieldName: string,
+      fieldValue: string,
+      inline?: boolean
+    ): void;
+    error(
+      title: string,
+      fieldName: string,
+      fieldValue: string,
+      inline?: boolean
+    ): void;
+  }
+  export type WebhookField = {
+    name?: string;
+    value?: string;
+    inline?: boolean;
+  };
+  export type WebhookEmbed = {
+    author?: {
+      name: string;
+      url: string;
+      icon_url: string;
+    };
+    title?: string;
+    url?: string;
+    thumbnail?: {
+      url: string;
+    };
+    image?: {
+      url: string;
+    };
+    timestamp?: Date;
+    color?: number;
+    description?: string;
+    fields: WebhookField[];
+    footer?: {
+      text?: string;
+      icon_url?: string;
+    };
+  };
+
+  export type WebhookPayload = {
+    embeds: WebhookEmbed[];
+  };
+
+  export class MessageBuilder {
+    constructor();
+    getJson(): WebhookPayload;
+    setText(text: string): this;
+    setAuthor(author: string, authorImage?: string, authorUrl?: string): this;
```

Comment:
Name can be optional.
https://discord.com/developers/docs/resources/channel#embed-object-embed-author-structure
```suggestion
    setAuthor(author?: string, authorImage?: string, authorUrl?: string): this;
```